### PR TITLE
Feature/int 34 submit review

### PR DESCRIPTION
**Pull request by rkaczka-vd**
this contains @addy419 ['s changes](https://github.com/IndicoDataSolutions/indico-client-csharp/pull/24)

Integration tests require environment with workflow with enabled autoreviews.

### CODE REVIEW

**Review by goatrocks — APPROVED** [review]

File: IndicoV2.IntegrationTests/Utils/DataHelpers/Workflows/WorkflowHelper.cs [hunk 1]
```diff
@@ -19,10 +25,8 @@ public async Task<IWorkflow> GetAnyWorkflow()
             return workflows.First();
         }
 
-        public WorkflowHelper(IDataSetClient dataSets, IWorkflowsClient workflows)
-        {
-            _dataSets = dataSets;
-            _workflows = workflows;
-        }
+        public async Task<IWorkflow> Get(bool autoReviewEnabled) => 
+            // TODO: implement autoReviewEnabled
```
Comment (szymon-socha) [inline IndicoV2.IntegrationTests/Utils/DataHelpers/Workflows/WorkflowHelper.cs:29]: Should this be implemented?

File: IndicoV2.Abstractions/Reviews/IReviewsClient.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+{
+    public interface IReviewsClient
+    {
+        /// <summary>
+        /// Submits review for <see cref="ISubmission"/>
+        /// </summary>
+        /// <param name="submissionId"></param>
+        /// <param name="changes"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
```
Comment (szymon-socha) [inline IndicoV2.Abstractions/Reviews/IReviewsClient.cs:16]: We should provide some comments.

File: IndicoV2.Abstractions/Reviews/IReviewsClient.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        Task<string> SubmitReviewAsync(int submissionId, JObject changes, CancellationToken cancellationToken);
+        
+        /// <summary>
+        /// Rejects <see cref="ISubmission"/>.
+        /// </summary>
+        /// <param name="submissionId"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
```
Comment (szymon-socha) [inline IndicoV2.Abstractions/Reviews/IReviewsClient.cs:24]: Same here.

File: IndicoV2.Abstractions/Reviews/IReviewsClient.cs (hunk 2 above)
Comment (rkaczka-vd) [reply IndicoV2.Abstractions/Reviews/IReviewsClient.cs:16]: fixing

File: IndicoV2.IntegrationTests/Utils/DataHelpers/Workflows/WorkflowHelper.cs (hunk 1 above)
Comment (rkaczka-vd) [reply IndicoV2.IntegrationTests/Utils/DataHelpers/Workflows/WorkflowHelper.cs:29]: This should be done after we have CI working (this method should be able to return workflow with autoReviewEnabled=true).

File: IndicoV2.Abstractions/Reviews/IReviewsClient.cs (hunk 3 above)
Comment (rkaczka-vd) [reply IndicoV2.Abstractions/Reviews/IReviewsClient.cs:24]: fixing

**Pull request merged**